monkey/monkey_island/cc/models/machine.py

```diff
@@ -12,12 +12,25 @@
 
 
 class Machine(MutableInfectionMonkeyBaseModel):
+    """Represents machines, VMs, or other network nodes discovered by Infection Monkey"""
+
     id: MachineID = Field(..., allow_mutation=False)
+    """Uniquely identifies the machine within the island"""
```

Comment:
Is this the correct way to document the fields  ? Probably we will need [this](https://pypi.org/project/autodoc-pydantic/)

Reply:
The `autodoc-pydantic` project you linked to is the best thing I could find on the subject.